### PR TITLE
Add BLXq Testnet

### PR DESCRIPTION
**Pull request by BLXDev20**
Add BLXq Testnet

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
explorer seems incompatible

File: _data/chains/eip155-1107.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "infoURL": "https://blx.org",
+  "shortName": "blxq",
+  "chainId": 1107,
+  "networkId": 1107,
+  "explorers": [
+    {
+      "name": "BLXq Explorer",
+      "url": "https://explorer.blx.org",
+      "icon": "blxq",
+      "standard": "EIP3091"
```
[the remark this answers is not in the document]
Comment (surgarshorts) [reply _data/chains/eip155-1107.json]: Am I suppose to change this? I'm still learning my way, I get confused & don't understand everything.

**Comment by BLXDev20** [comment]
Hi @ligi can you look at this. my icon is valid:

https://explore.ipld.io/#/explore/QmS9kDKr1rgcz5W55yCQVfFs1vRTCneaLHt1t9cBizpqpH

Thanks

**Review by ligi — APPROVED** [review]

**Review by jonesk7734 — APPROVED** [review]
Ok

**Pull request merged**